host/tests/.gitignore

```diff
@@ -0,0 +1 @@
+raft
```

Comment:
Probably should add ut_raft in here too?

Reply:
I added the plugins directory as a change to raft, rather than making it project specific here.